src/ensemble_scheduler.cc

```diff
@@ -340,6 +350,10 @@ class EnsembleContext {
   // Objects related to the ensemble infer request
   Status ensemble_status_;
   RequestTracker* request_tracker_;
+  // Use in conjunction with 'is_decoupled_' in EnsembleInfo to
+  // better distinguish ensemble ending behavior (see documentation in
+  // FinishEnsemble for detail).
```

Comment:
My only complain here is that I was expecting to see **documentation**. I think `annotation` would be better for current case